tap_lever/streams/opportunities.py

```diff
@@ -10,8 +18,122 @@ class OpportunityStream(TimeRangeStream):
     TABLE = "opportunities"
     KEY_PROPERTIES = ["id"]
 
-    CACHE_RESULTS = True
-
     @property
     def path(self):
         return "/opportunities"
+
+    def sync(self, child_streams=None):
+        LOGGER.info('Syncing stream {} with {}'
+                    .format(self.catalog.tap_stream_id,
+                            self.__class__.__name__))
+
+        self.write_schema()
+
+        return self.sync_data(child_streams)
+
+    def sync_paginated(self, url, params=None, child_stream=None):
+        table = self.TABLE
+        _next = True
+        page = 1
+
+        all_resources = []
+        transformer = singer.Transformer()
+        while _next is not None:
```

Comment:
Maybe just `while next`? Is "False" a "continue" kind of value?

Reply:
Ehhhh... if for some reason a value of `''` or `False` is valid for `_next`, it might be best not to touch this.

Reply:
Avoiding!